frontend/src/lib/forms/Input.svelte

```diff
@@ -6,27 +6,26 @@
   export let label: string;
   export let description: string | undefined = undefined;
   export let value: string | undefined = undefined;
-  export let type = 'text';
+  export let type: 'text' | 'email' | 'password' = 'text';
```

Comment:
should we put a comment here that non string types should not be used as they won't bind properly?

Reply:
If someone tried to extend the type of `value` to include `number` they'd run into serious type trouble and decide that it's a terrible idea. So, I don't think a comment is necessary. If they just added `number` to the `type`, they could go on their merry way and would just miss out on a tiny bit of Svelte-magic like most web developers do 😉.